modules/mod_breadcrumbs/tmpl/default.php

```diff
@@ -78,14 +78,30 @@
 
 	foreach ($list as $key => $item)
 	{
-		$data['itemListElement'][] = [
-				'@type'    => 'ListItem',
-				'position' => $key + 1,
-				'item'     => [
-						'@id'  => $item->link ? Route::_($item->link, true, Route::TLS_IGNORE, true) : Route::_(Uri::getInstance()),
-						'name' => $item->name
-				]
-		];
+		// Only add item to JSON if it has a valid link, otherwise skip it.
+		if (!empty($item->link))
+		{
+			$data['itemListElement'][] = [
+					'@type'    => 'ListItem',
+					'position' => $key + 1,
+					'item'     => [
+							'@id'  => $item->link ? Route::_($item->link, true, Route::TLS_IGNORE, true) : Route::_(Uri::getInstance()),
+							'name' => $item->name,
+					],
+			];
+		}
+		elseif ($key === $last_item_key)
+		{
+			// Add the last item (current page) to JSON, but without a link.
+			// Google accepts items without a URL only as the current page.
+			$data['itemListElement'][] = [
+					'@type'    => 'ListItem',
+					'position' => $key + 1,
+					'item'     => [
+							'name' => $item->name
+					]
```

Comment:
```suggestion
							'name' => $item->name,
					],
```
Sorry, this one I have forgotten in my last review. Of course also here for the last array elements a comma at the end due to our code style rules.